dipy/viz/tests/test_ui.py

```diff
@@ -646,6 +803,15 @@ def test_ui_image_container_2d(interactive=False):
     if len(sys.argv) <= 1 or sys.argv[1] == "test_ui_range_slider":
         test_ui_range_slider(interactive=False)
 
+    if len(sys.argv) <= 1 or sys.argv[1] == "test_ui_option":
+        test_ui_option(interactive=False)
+
+    if len(sys.argv) <= 1 or sys.argv[1] == "test_ui_checkbox":
+        test_ui_checkbox(interactive=False)
+
+    if len(sys.argv) <= 1 or sys.argv[1] == "test_ui_radio_button":
+        test_ui_radio_button(interactive=False)
```

Comment:
It should be `interactive=True` though. When the script is called directly, interactive should be true.